resources/lang/de_DE/default.po

```diff
@@ -539,6 +539,9 @@ msgstr "Nr."
 msgid "general.shifts"
 msgstr "Schichten"
 
+msgid "general.my_shifts"
+msgstr "Meine Schichten"
+
```

Comment:
```suggestion
```
try moving the existing translation from `additional.po` here

Reply:
would you keep the `profile.` prefix from `additional.po` (as far as I can see there's no other item with this exact prefix) or would you convert current uses of `profile.my_shifts`to `general.my_shifts` – or pick a different prefix entirely?

Reply:
just move them from `additional.po` to `default.po`